riak/transports/pbc/codec.py

```diff
@@ -85,13 +85,7 @@ def __init__(self, **unused_args):
 
     def _unix_time_millis(self, dt):
         td = dt - epoch
-        try:
-            return int(dt.total_seconds() * 1000.0)
-        except AttributeError:
-            # NB: python 2.6 must use this method
-            return int(((td.microseconds +
-                         (td.seconds + td.days * 24 * 3600) * 10**6) /
-                        10**6) * 1000.0)
+        return int(td.total_seconds() * 1000.0)
```

Comment:
You'll notice there was a typo previously where `dt.total_seconds()` was being called, instead of `td.total_seconds()`. This triggered the `AttributeError` and the logic in the exception handler would discard milliseconds.